pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -237,6 +237,23 @@ def test_build_series(self):
 
         assert result == expected
 
+    def test_read_json(self):
+        # GH32383
+        df = pd.DataFrame(
+            {
+                "_id": {"0": "0"},
+                "category": {"0": "Goods"},
+                "recommender_id": {"0": "3"},
+                "recommender_name_en": {"0": "Urata"},
+                "name_en": {"0": "Hakata Dolls Matsuo"},
```

Comment:
Could you include some of the ASCII characters that were in the original issue?

Reply:
sure